cmd/dependabot/internal/cmd/update_test.go

```diff
@@ -147,12 +148,22 @@ func Test_extractInput(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		defer os.Remove(tmp.Name())
+		t.Cleanup(func() { os.Remove(tmp.Name()) })
 
-		tmp.WriteString(`{"job":{"package-manager":"go_modules"}}`)
+		_, err = tmp.WriteString(`{"job":{"package-manager":"go_modules"}}`)
+		if err != nil {
+			t.Fatal(err)
+		}
```

Comment:
All these three liners just to check errors that are expected 99% of the time to be nil are killing my eyes, would much prefer `testify`'s:
```go
require.NoError(t, err)
```

And be done with it...